docs/index.md

```diff
@@ -24,38 +24,24 @@ See the relationship between the connector files (in blue) and the Tableau **Con
 
 ![]({{ site.baseurl }}/assets/files-overview.png)
 
-## Before you begin
+# What is a taco?
```

Comment:
Thanks for doing this!

:27 uses `taco` and :28 uses `.taco`. We should probably standardize how and when we use each term. 

To spitball:  
`.taco` for referring to the file itself 
_taco_ to refer to a packaged connector.

What do we think about **Taco** vs **taco** for the latter case?

It may be in the docs already, but do we want to explain why it's called a taco?

And for fun I think we should include the cool .taco graphic somewhere 😉

Reply:
Hmm, good question. I'll go with taco (no dot) for now. I'll also make the capilization for the headers in this file title style, in line with the other headers in other files.

Reply:
FYI - In our other external documentation, we use the convention of tagging the file name extensions in with `code` tags and we also include the `.` .  For example, `.trex` for Dashboard Extensions.   So you probably want to use `.taco` when you are just talking about the file. Calling Tableau Connectors "Tacos" seems pretty cool. I'd use title case for that "Taco"
Check with Kathy when she gets back.   I believe she is planning on using the taco.png file on the landing page.

Reply:
Formatted the the extension names to be `.taco`, and when not talking about the file capitalized it.